robyn/__main__.py

```diff
@@ -74,7 +79,11 @@ def index():
 
 def docs():
     print("Opening Robyn documentation... | Offline docs coming soon!")
-    webbrowser.open("https://sansyrox.github.io/robyn/#/")
+    webbrowser.open("https://robyn.tech/")
```

Comment:
One final suggestion - can we add this link instead for now ? https://sparckles.github.io/robyn/#/

Reply:
Awesome we can go for it... 🕺🏻